packages/fiori/src/ShellBar.ts

```diff
@@ -29,387 +49,49 @@ import {
-		 *
-		 * @type {sap.ui.webc.main.IInput}
-		 * @slot
-		 * @public
-		 */
-		searchField: {
-			type: HTMLElement,
-		},
-
-		/**
-		 * Defines a <code>ui5-button</code> in the bar that will be placed in the beginning.
-		 * We encourage this slot to be used for a back or home button.
-		 * It gets overstyled to match ShellBar's styling.
-		 *
-		 * @type {sap.ui.webc.main.IButton}
-		 * @slot
-		 * @public
-		 */
-		startButton: {
-			type: HTMLElement,
-		},
-	},
-	events: /** @lends sap.ui.webc.fiori.ShellBar.prototype */ {
-		/**
-		 *
-		 * Fired, when the notification icon is activated.
-		 *
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#notifications-click
-		 * @allowPreventDefault
-		 * @param {HTMLElement} targetRef dom ref of the activated element
-		 * @public
-		 */
-		"notifications-click": {
-			detail: {
-				targetRef: { type: HTMLElement },
-			},
-		},
-
-		/**
-		 * Fired, when the profile slot is present.
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#profile-click
-		 * @param {HTMLElement} targetRef dom ref of the activated element
-		 * @public
-		 */
-		"profile-click": {
-			detail: {
-				targetRef: { type: HTMLElement },
-			},
-		},
-
-		/**
-		 * Fired, when the product switch icon is activated.
-		 * <b>Note:</b> You can prevent closing of overflow popover by calling <code>event.preventDefault()</code>.
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#product-switch-click
-		 * @allowPreventDefault
-		 * @param {HTMLElement} targetRef dom ref of the activated element
-		 * @public
-		 */
-		"product-switch-click": {
-			detail: {
-				targetRef: { type: HTMLElement },
-			},
-		},
-
-		/**
-		 * Fired, when the logo is activated.
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#logo-click
-		 * @param {HTMLElement} targetRef dom ref of the activated element
-		 * @since 0.10
-		 * @public
-		 */
-		"logo-click": {
-			detail: {
-				targetRef: { type: HTMLElement },
-			},
-		},
-
-		/**
-		 * Fired, when the co pilot is activated.
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#co-pilot-click
-		 * @param {HTMLElement} targetRef dom ref of the activated element
-		 * @since 0.10
-		 * @public
-		 */
-		"co-pilot-click": {
-			detail: {
-				targetRef: { type: HTMLElement },
-			},
-		},
-
-		/**
-		 * Fired, when a menu item is activated
-		 * <b>Note:</b> You can prevent closing of overflow popover by calling <code>event.preventDefault()</code>.
-		 *
-		 * @event sap.ui.webc.fiori.ShellBar#menu-item-click
-		 * @param {HTMLElement} item DOM ref of the activated list item
-		 * @since 0.10
-		 * @public
-		 */
-		"menu-item-click": {
-			detail: {
-				item: { type: HTMLElement },
-			},
-		},
-	},
+type AccessibilityAttributes = {
+	notifications?: AccessibilityAttributesProperties;
+	profile?: AccessibilityAttributesProperties;
+	product?: AccessibilityAttributesProperties;
+	search?: AccessibilityAttributesProperties;
+	overflow?: AccessibilityAttributesProperties;
 };
 
+interface IItem {
```

Comment:
"IItem"  is too generic, you better relate it to the ShellBarItem, probably ShelBarItemInfo or something  along these lines.